flang/lib/Lower/OpenMP/OpenMP.cpp

```diff
@@ -692,53 +692,102 @@ static void createBodyOfOp(mlir::Operation &op, const OpWithBodyGenInfo &info,
   marker->erase();
 }
 
+void mapBodySymbols(lower::AbstractConverter &converter, mlir::Region &region,
+                    mlir::Block *regionBlock,
```

Comment:
Nit: Could this argument be removed and just access it from `region.front()`? This function is expected to be called while building the entry block, so we can just `assert(region.hasOneBlock())`.